docker/dependencies.repos

```diff
@@ -0,0 +1,9 @@
+repositories:
+  cartographer/cartographer_ros:
+    type: git
+    url: https://github.com/ROBOTIS-GIT/cartographer_ros.git
+    version: crystal
+  navigation2/navigation2:
+    type: git
+    url: https://github.com/crdelsey/navigation2.git
+    version: 768-cmake-fix
```

Comment:
Using custom repo/branch until https://github.com/ros-planning/navigation2/pull/860 is merged as the nav2 packages don't compile without it